src/me.rs

```diff
@@ -167,8 +167,7 @@ mod nasm {
 
   #[inline(always)]
   pub fn get_sad<T: Pixel>(
-    plane_org: &PlaneSlice<'_, T>, plane_ref: &PlaneSlice<'_, T>, blk_h: usize,
-    blk_w: usize, bit_depth: usize
+    plane_org: &PlaneSlice<'_, T>, plane_ref: &PlaneSlice<'_, T>, blk_w: usize, blk_h: usize,bit_depth: usize
```

Comment:
You also need to switch the parameters for all callers.

Reply:
cool :+1: